src/com/untamedears/PrisonPearl/PrisonPearlCommands.java

```diff
@@ -177,10 +180,34 @@ public boolean onCommand(CommandSender sender, Command cmd, String label, String
         	return banAlt(sender, args);
         } else if (label.equalsIgnoreCase("ppunban")){
         	return unBanAlt(sender, args);
+        } else if (label.equalsIgnoreCase("ppwb")){
+        	if(sender.hasPermission("prisonpearl.wb")) {// sees if the players has the permission.
+                return WhiteListedLocations(sender, args);}
+            else{ sender.sendMessage("You Do not have Permissions prisonpearl.wb");}// if players doesn't have permission, broadcasts message saying what they are missing.
         }
         return false;
     }
 
+	private boolean WhiteListedLocations(CommandSender sender, String[] args) {
+		if (args.length != 4)
+    	{
```

Comment:
Again nitpicking, sorry, but your brackets are stranded on their own lines and indent levels. :(